route/account/_id/index.js

```diff
@@ -0,0 +1,45 @@
+"use strict";
+
+module.exports = (server, path, options) => {
+    return options.container.call((accountManager, config, sessionManager) => {
+        var loginCookie;
+
+        loginCookie = require("./_login-cookie")(config);
```

Comment:
Directories are flagged as parameters and are special when prefixed with an underscore.  I needed a way to have route-specific libraries and thought they should live in the `routes/` folder instead of having them live in `lib/`.  It felt cleaner to me.  Perhaps route-related libraries could be moved to another location in the future.

Anyway, if you look at `lib/web-server.js` you will see that I exclude these files from being served as routes with the `!(_)` part of the `routesMatch` option when configuring restify-router-magic.